x-pack/legacy/plugins/lens/public/indexpattern_plugin/dimension_panel/settings.tsx

```diff
@@ -53,59 +41,166 @@ export function Settings({
         );
       })
     : filteredColumns;
+  const possibleOperationTypes = filteredColumns.map(col => ({
+    operationType: col.operationType,
+    compatibleWithCurrentField: false,
+  }));
+  const validOperationTypes = columnsFromField.map(col => ({
+    operationType: col.operationType,
+    compatibleWithCurrentField: true,
+  }));
+  return _.uniq([...validOperationTypes, ...possibleOperationTypes], 'operationType');
+}
+
+export interface SettingsProps extends IndexPatternDimensionPanelProps {
+  selectedColumn?: IndexPatternColumn;
+  filteredColumns: IndexPatternColumn[];
+}
+
+export function Settings(props: SettingsProps) {
```

Comment:
I wasn't really sure what this `Settings` name referred to. I might rename this to `FlyoutContents` or `FlyoutEditor` to indicate how it's used

Reply:
you are right, I renamed to `PopoverEditor` - flyout is something else in EUI world (https://elastic.github.io/eui/#/layout/flyout)